### PR TITLE
[serverless-1.30.0] Patch tekton pipeline task reference

### PR DESCRIPTION
**Pull request by jrangelramos**
Change required  for Pipelines as Code to reference midstream repository

### CODE REVIEW

File: pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml [hunk 1]
```diff
@@ -0,0 +1,234 @@
+---
```
Comment (matejvasek) [inline pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: why this leading `---` ?

Comment (jrangelramos) [reply pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: yaml notation

Comment (matejvasek) [reply pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: Oh I see it is also in orig file. Weird.

Comment (matejvasek) [reply pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: that is used for multi doc yamls, here we have single resource defined

Comment (matejvasek) [reply pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: The thing is: some parser refuse to parse multi doc yaml files

Comment (jrangelramos) [reply pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: hmmm.. I see.. yea.. we can remove it.. but that *pac file is just a sole copy of the original one, except for the ClusterTask to Task change.. .that's on upstream as well https://raw.githubusercontent.com/knative/func/main/pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml

Comment (matejvasek) [reply pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml:1]: I think it's fine now.

**Comment by matejvasek** [comment]
@jrangelramos some of the tests are failing because of Go version.

**Comment by matejvasek** [comment]
@jrangelramos you need to downgrade Go to 1.20.5 in GH workflows.

**Comment by matejvasek** [comment]
this might meed override

**Pull request merged**